glotaran/builtin/models/kinetic_spectrum/kinetic_spectrum_result.py

```diff
@@ -11,59 +8,30 @@
 from glotaran.builtin.models.kinetic_image.kinetic_image_result import (
     retrieve_species_assocatiated_data,
 )
-from glotaran.parameter import ParameterGroup
 
-from .spectral_constraints import OnlyConstraint
-from .spectral_constraints import ZeroConstraint
 from .spectral_irf import IrfGaussianCoherentArtifact
 from .spectral_irf import IrfSpectralMultiGaussian
 
+T_Problem = typing.TypeVar("glotaran.analysis.problem.Problem")
```

Comment:
This will cause 'not valid as a type' see #456 for a reference how it should be done.

Reply:
I fixed this one myself after rebasing